File: applied/05_top_of_the_world/answer/3.c
// 1から100まで順番に数字を数えていき、
// その数字が3の倍数であれば "Hello",
// その数字が5の倍数であれば "World",
// 15の倍数であれば "HelloWorld",
// それ以外であればその数字をそのまま出力するプログラムを作成しよう

#include <stdio.h>
int main(void){
    int i;

    for (i = 1; i <= 100; i++) {
        if (i % 15 == 0) {
            printf("HelloWorld\n");
        } else if (i % 3 == 0) {
            printf("Hello\n");
        } else if (i % 5 == 0) {
            printf("World\n");
        } else {
            printf("%d\n", i);
        }
    }
}

File: applied/15_string/answer/_3.java
// 変数に "Hello world" を格納し、出力してください

import java.util.*;

public class _3 {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        String input;

        for (input = scan.nextLine(); input.length() >= 10; input = scan.nextLine()) ;
        
        System.out.printf("%s\n", input);
        scan.close();
    }
}
